assignments/arrays.js

```diff
@@ -63,34 +63,65 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+for(let i = 0; i < inventory.length; i++){
+    if(inventory[i]["id"] === 33){
+        console.log(`Car ${inventory[i]["id"]} is a ${inventory[i]["car_year"]} ${inventory[i]["car_make"]} ${inventory[i]["car_model"]}` );
+    }
+}
+
+
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[inventory.length - 1];
+console.log(`${lastCar["car_make"]} ${lastCar["car_model"]}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for(let i = 0 ; i < inventory.length; i++){
+    carModels.push(inventory[i]["car_model"]);
+}
+carModels.sort();
+
+for(let i = 0 ; i < carModels.length; i++){
+    console.log(carModels[i]);
+}
+
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for(let i = 0; i < inventory.length; i++){
+    carYears.push(inventory[i]["car_year"]);
+}
+for(let i = 0; i < carYears.length; i++){
+    console.log(carYears[i]);
+}
```

Comment:
You could log the cars as you push them here (so that you don't have to loop twice):
```javascript
for(let i = 0; i < inventory.length; i++){
    carYears.push(inventory[i]["car_year"]);
    console.log(carYears[i]);
}
```